src/app/component/ProductDetails/ProductDetails.style.scss

```diff
@@ -59,6 +59,24 @@
         }
     }
 
+    &-ReviewSummary {
+        font-size: 1.1rem;
+        margin-top: .5rem;
+
+        .ProductReviewRating,
+        a {
+            display: inline-block;
+        }
+
+        a {
+            vertical-align: sub;
```

Comment:
Are you sure about this? Is this a work-around, or intended?

Reply:
This one is used to better align rating stars with review count text. Tried other options but this one provided the best alignment between them.